### ISSUE TITLE
Add aria props and instructions to docs about accessibility

### ISSUE THREAD

**Issue by juliankrispel** [juliankrispel/react-text-selection-popover#12]

**Comment by sunilhari** [comment]
Hi  Julian
so it should have `aria-expanded` ,`aria-haspopover` and `aria-hidden` tags.

Is my understanding correct @juliankrispel

**Comment by juliankrispel** [comment]
@sunilhari yes kind of, the parent component should have the first two, the popover itself should use aria-hidden. But really it needs to just be tested ...
